prog/vm/github_runner.rb

```diff
@@ -71,17 +71,10 @@ def pick_vm
       boot_image: label_data["boot_image"],
       storage_volumes: [vm_storage_params],
```

Comment:
does this commit message really say what's going on? I'm a bit puzzled what's being removed (it's presumed obsolete) and I guess relying on allow_only_ssh which was deployed earlier. It's more like deprecating one usage for another?

Reply:
I have updated the commit message